contributing/code/patches.rst

```diff
@@ -1,10 +1,19 @@
-Submitting a Patch
-==================
+Submitting a Change
+===================
 
-Patches are the best way to provide a bug fix or to propose enhancements to
-Symfony.
+A pull request, "PR" for short, is the best way to provide a bug fix or to propose
+enhancements to Symfony.
 
-Step 1: Setup your Environment
+Step 1: Check existing issues and PRs
+-------------------------------------
+
+Before working on a change, check to see if someone else also raised the topic
+or maybe even started working on a PR by `searching on GitHub`_.
+
+If you are still unsure or if you have any questions during this entire process,
```

Comment:
I would remove "still" here. At the moment, it sort of looks like you have to ask on slack if you didn't find a related issue?